modules/ROOT/pages/release-notes/release-notes501.adoc

```diff
@@ -10,4 +10,4 @@ TinyMCE 5.0.1 release adds improvements to the TinyMCE editor to improve the ove
 
 == Updates and enhancements
 
-The new TinyMCE 5.0.1 editor comes with significant changes to the previous versions. Tiny has created a more configurable and modern editor. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the link:{baseurl}/changelog/#version501february212019[Changelog].
+The new TinyMCE 5.0.1 editor comes with significant changes to the previous versions. Tiny has created a more configurable and modern editor. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the xref:changelog.adoc#version501february212019[Changelog].
```

Comment:
This xref doesn't resolve as the `version501february212019` id doesn't exist on the relevant title in `changelog.adoc` (well technically it's using the old `a` elements). e.g:
```
[[version501february212019]]
== 5.0.1 - 2019-02-21
```

Reply:
done